packages/nextjs/components/Header.tsx

```diff
@@ -90,11 +90,11 @@ export const Header = () => {
         </div>
         <Link href="/" passHref className="hidden lg:flex items-center gap-2 ml-4 mr-6 shrink-0">
           <div className="flex relative w-10 h-10">
-            <Image alt="SE2 logo" className="cursor-pointer" fill src="/logo.svg" />
+            <Image alt="Block #3 logo" className="cursor-pointer" fill src="/block3-logo.png" />
```

Comment:
Maybe we should keep the alt as "Batch #3 logo" instead of "Block ..." 

Lol I am not sure if it was intended, I see we are using "Block"  instead of "Batch"  or "Blob"

Reply:
Oh, good catch, thanks!